composer.json

```diff
@@ -51,5 +54,10 @@
     },
     "scripts": {
         "test": "@php vendor/bin/phpunit --colors=always"
+    },
+    "config": {
```

Comment:
This should be removed, it will be added again after install, but we need to ignore it.